client/tests/gui/test_widgets.py

```diff
@@ -3334,7 +3334,9 @@ def test_FileWidget_on_left_click_download(mocker, session, source):
 
     fw._on_left_click()
     get_file.assert_called_once_with(file_.uuid)
-    controller.on_submission_download.assert_called_once_with(db.File, file_.uuid)
+    # Because the ProgressProxy is created dynamically and not retained, we can't assert
+    # the specific value of it, so use ANY.
```

Comment:
(This is true - optionally you could inspect the call_args and ensure that the thing that was passed in the 3rd arg is of type ProgressProxy)